app/models/batch_invitation_user.rb

```diff
@@ -5,9 +5,9 @@ class BatchInvitationUser < ActiveRecord::Base
 
   validates :outcome, inclusion: { :in => [nil, "success", "failed", "skipped"] }
 
-  scope :processed, where("outcome IS NOT NULL")
-  scope :unprocessed, where("outcome IS NULL")
-  scope :failed, where(outcome: "failed")
+  scope :processed, -> { where("outcome IS NOT NULL") }
+  scope :unprocessed, -> { where("outcome IS NULL") }
```

Comment:
Is it worth updating these to the new syntax now, `where.not(outcome: nil)` and `where(outcome: nil)`?